src/misc/crypt/crypt_ltc_mp_descriptor.c

```diff
@@ -8,9 +8,20 @@
  */
 #include "tomcrypt.h"
 
+<<<<<<< HEAD
```

Comment:
@t0p4 Please beautify the history, there is still commits with conflicts.

Reply:
It was resolved

Reply:
Commit https://github.com/hunter-packages/libtomcrypt/pull/3/commits/44e904f7ad2c4e03ee47b0f3b4d5c0a6ce7751a8 is a part of pull request and there is a conflict in it. It will be quite messy to use this commit in future for adapting next version, please beautify the history.

Reply:
How do it more beautify ?

Reply:
Do you see that https://github.com/hunter-packages/libtomcrypt/commit/44e904f7ad2c4e03ee47b0f3b4d5c0a6ce7751a8 is a part of this pull request? Do you see that there is a conflict in https://github.com/hunter-packages/libtomcrypt/commit/44e904f7ad2c4e03ee47b0f3b4d5c0a6ce7751a8 ? Please make it so, that there will be no commits with `<<<<<<< HEAD`/`=======`/`>>>>>>>`. This is a purely git exercise, please check  some git tutorials/documentation for details.